### PR TITLE
Layout grid logic

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -6,7 +6,7 @@ function scripts_and_styles_method() {
   // library.js is to bundle plugins. my.js is your scripts. enqueue more files as needed
   $jslib = $templateuri."library.js";
   wp_enqueue_script( 'jslib', $jslib,'','',true);
-  $myscripts = $templateuri."main.js";
+  $myscripts = $templateuri."main.min.js";
```
Comment (mixcas) [inline functions.php:9]: WHY? we are not even half the way on this project…

Comment (varffs) [inline functions.php:9]: with sourcemaps there is no real need to ever serve a non minified file so better to have it like this all the way thru dev imo

Comment (mixcas) [inline functions.php:9]: debugging doesn't work well with min+sourcemap
if u wanna inspect thru the consoe it sux cuz  dont know the names of vars so u r lost
debugger will usually print next to each line the value of the vars used on that line, that doesn't work either

Comment (mixcas) [inline functions.php:9]: i was having an issue last night cuz of this in 8106 too…

Comment (varffs) [inline functions.php:9]: I think this is a sourcemap configuration issue though. To do with the combo of uglify and sourcemap

**Pull request merged**